src/tpm2.c

```diff
@@ -2177,9 +2196,10 @@ TPM_RC TPM2_Hash(Hash_In* in, Hash_Out* out)
 
     rc = TPM2_AcquireLock(ctx);
     if (rc == TPM_RC_SUCCESS) {
-        CmdInfo_t info = {0,0,0,0,};
-        info.flags = (CMD_FLAG_ENC2 | CMD_FLAG_DEC2);
         TPM2_Packet packet;
+        CmdInfo_t info = {0,0,0,0};
+        info.flags = (CMD_FLAG_ENC2 | CMD_FLAG_DEC2);
```

Comment:
Not critical, but here and elsewhere, why don't we just set the fields in the init of info? If we're going to assign them right after init to 0, might as well not init to 0 in the first place.

Reply:
This was changed to resolve a compiler issue: https://github.com/wolfSSL/wolfTPM/pull/206
Reverting my last commit.